WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureCelebration.swift

```diff
@@ -0,0 +1,30 @@
+import AudioToolbox
+import UIKit
+
+/// Plays a sound and provides haptic feedback when a payment capture has been completed successfully
+final class PaymentCaptureCelebration: NSObject {
+    private var soundID: SystemSoundID = 0
+
+    func celebrate() {
+        playSound()
+        shakeDevice()
+    }
+}
+
+private extension PaymentCaptureCelebration {
+    func playSound() {
+        guard let path = Bundle.main.path(forResource: "o.caf", ofType: nil) else {
+            return
+        }
+
+        let url = URL(fileURLWithPath: path)
+        AudioServicesCreateSystemSoundID(url as CFURL, &soundID)
```

Comment:
I think you're only meant to call this once for each sound in your application, or dispose of the system sound when you're done with it with `AudioServicesDisposeSystemSoundID(_:)`

Reply:
Nice catch thanks!